docs/deployment/manage-projects.md

```diff
@@ -51,6 +50,12 @@ The following filters are available:
 Scan details, such as logs, are available for scans run in the past **1 month**. Semgrep AppSec Platform does not display scan details older than 30 days, since this introduces performance issues due to the increased volume of stored scan data.
 :::
 
+## Run scans in bulk
+
+You can scan multiple projects at once from the **Projects** page. This is useful when you want to update multiple projects after changing your ruleset or configuration.
```

Comment:
```suggestion
You can scan multiple projects at once from the **Projects** page. This is useful when you want to scan multiple projects after changing your ruleset or configuration.
```

I know scan is a bit redundant, so feel free to rework, but update the project isn't quite right? Maybe "rescan" works?